crates/evm/core/src/backend/mod.rs

```diff
@@ -232,14 +233,16 @@ pub trait DatabaseExt: Database<Error = DatabaseError> + DatabaseCommit {
         inspector: &mut dyn InspectorExt,
     ) -> eyre::Result<()>;
 
+    /// Note(zk): This function in upstream code is not implemented as part of the strategy pattern.
```

Comment:
phrasing a bit confusing (for me), the note could just highlight that the type was changed to pass the data to the strategy, eg: `NOTE(zk): we changed the type signature to pass the raw data to the strategy directly`